dockerfiles/run/guice/cassandra/destination/conf/elasticsearch.properties

```diff
@@ -22,6 +22,10 @@
 
 elasticsearch.masterHost=elasticsearch
 elasticsearch.port=9200
+# Only http or https are accepted
+elasticsearch.hostScheme=https
+elasticsearch.user=elasticsearch
+elasticsearch.password=secret
```

Comment:
We may need to comment that by default...

Reply:
Should it be added in other elasticsearch.properties template files?